executor/adapter.go

```diff
@@ -821,6 +822,34 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, hasMoreResults bool) {
 	}
 }
 
+// getPlanTree will try to get the select plan tree if the plan is select or the select plan of delete/update/insert statement.
+func getPlanTree(p plannercore.Plan) string {
```

Comment:
There is some difference between `getPlanTree` with `Explain.RenderResult()`:
* Use Plan ID instead of Plan name in the plan tree.
* Use `snappy` and `base64` to encode plan tree to a string.